content/en/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -221,14 +221,15 @@ use the LocalService service account as it is the least privileged of the three
 
 ### Local accounts {#local-accounts}
 
-HostProcess containers can also run as local user accounts which allows for node operators to give
+If configured, HostProcess containers can also run as local user accounts which allows for node operators to give
 fine-grained access to workloads.
 
 To run HostProcess containers as a local user; A local usergroup must first be created on the node
 and the name of that local usergroup must be specified in the `runAsUserName` field in the deployment.
-This will cause an new ephemeral local user account to be created, joined to the specified usergroup,
-and used by the container. This provides a number a benefits including eliminating the need to manage
-passwords for local user accounts.
+Prior to initializing the HostProcess container, a new **ephemeral** local user account to be created and joined to the specified usergroup, from which the container is run.
```

Comment:
to be really clear here, might say something like "the runtime will create an ephemeral".  

", from which the container is run." is a bit award to read I think.